cmd/builder/builder_options_test.go

```diff
@@ -36,7 +37,12 @@ func TestApplyOptions(t *testing.T) {
 		Options.LoggerOption(zap.NewNop()),
 		Options.MetricsFactoryOption(metrics.NullFactory),
 		Options.MemoryStoreOption(memory.NewStore()),
+		Options.ElasticSearchOption(&escfg.Configuration{
+			Servers: []string{"127.0.0.1"},
```

Comment:
https://github.com/uber/jaeger/blob/master/cmd/query/app/builder/storage.go#L52

I think a later goal is to allow multiple storage for different components of jaeger; i.e. dependencies and spans, and in this case, we would need multiple storage options.